components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildScanDataCommand.java

```diff
@@ -174,17 +174,43 @@ private void logFetchResultFor(int runNum, String property, String customValueKe
         );
     }
 
-    public void printHeader() {
-        List<String> labels = Fields.ordered().map(f -> f.label).collect(Collectors.toList());
+    private static void printResults(List<BuildScanData> buildScanData) {
+        printBuildScanDataHeader();
+        printBuildScanData(buildScanData);
+
+        if (buildScanData.size() == 2) {
+            printBuildTimeMetricsHeader();
+            printBuildTimeMetrics(buildScanData);
+        }
+    }
+
+    public static void printBuildScanDataHeader() {
+        List<String> labels = BuildScanDataFields.ordered().map(f -> f.label).collect(Collectors.toList());
         System.out.println(String.join(",", labels));
```

Comment:
This could be simplified slightly by using [Collectors.joining](url):

```java
System.out.println(BuildScanDataFields.ordered().map(f -> f.label).collect(Collectors.joining(",")));
```

Reply:
Resolving as this was done in #391